### ISSUE TITLE
Module parse failed: Unexpected token (1:0)

### ISSUE THREAD

**Issue by spevilgenius** [pnp/generator-spfx#196]
#### General Information
* [*] Usage
* [*] Development
* [ ] Documentation
* [ ] Feature Request

#### Specific generator
* [ ] Addons
* [ ] HandlebarsJS

#### What Version you are running?
* NodeJS: 10.16.3
* NPM: 6.11.2
* @pnp/spfx: 1.9.0

#### Describe your problem

Clean install target is spo and gulp serve produces error:
Module parse failed: Unexpected token (1:0)
You may need an appropriate loader to handle this file type.

<template>
|     <div :class="$style.HelloWorld">
|         <div :class="$style.container">
 @ ./lib/webparts/helloWorld/HelloWorldWebPart.js 17:0-62 29:44-63

I tried to see if this had been reported but may have missed it. Just tried this today. Really trying to get Vue going!!

**Comment by StfBauer** [comment]
@dpevilgenius - Can you please share what options you have chosen for scaffolding the project?
Screenshot of all options selected will do.
Thanks 🙏

**Comment by spevilgenius** [comment]
![Capture](https://user-images.githubusercontent.com/20232725/63960907-901b7a80-ca5d-11e9-9ff6-4777aa4ee6ae.JPG)

I mostly chose defaults and targeted spo

**Comment by StfBauer** [comment]
There is one strange thing in your screenshot. It seems to me like you never have been asked for the Framework to choose? And not even the other questions. 

Can you please check if a .yo-rc file already in the folder exists? The create a subfolder and .yo-rc file mostly have some negative side effects. If there is a .yo-rc file already can you please share the content?

These are the questions from my test run on a blank and completely empty folder:
<img width="1247" alt="Screenshot 2019-08-29 at 20 16 24" src="https://user-images.githubusercontent.com/5503835/63965703-08079600-ca9a-11e9-872f-529fb000aca2.png">

The best way is no matter if you use the Microsoft or this generator to start from a blank folder and leave do not create a subfolder. Thank you for help to understand what causes this issue.

**Comment by spevilgenius** [comment]
Yes, I didn't get most of those choices!! There was a .yo-rc file in the Repos folder but no idea how it got there!! Removed it and voila! Success! Well, mostly. I do receive:
TS5069: Option 'declarationMap' cannot be specified without specifying option 'declaration' or option 'composite'. 

But I can still serve and test so that's good!!

**Comment by StfBauer** [comment]
Great @spevilgenius !!! You always get to `.yo-rc`files when you choose the create folder option. On in the folder you started and one exactly the same in the solution folder itself. Might some leftover.

What TypeScript generator have you chosen? That shows you this message.

**Comment by spevilgenius** [comment]
That message was just when I ran gulp serve. I chose the 3.5 version of TypeScript but that's it. To be honest, I wish I could do it without TypeScript at all. Just so used to the vue cli and javascript files without the added weight of TypeScript. Just my 2 cents!

**Comment by StfBauer** [comment]
Yeah, understandable I currently think about maybe some future options to make the experience better.
